global/css/plugins/theme/colors.js

```diff
@@ -7,5 +7,6 @@ module.exports = function () {
     "dark-mode": {
       red: "#EC0909",
     },
+    'campaign-fog': "#F9F9F9",
```

Comment:
I thought the background light grey was `fog-light` at first, but it's a slightly lighter grey!